test/kube_testing/pods/nsmd.go

```diff
@@ -19,15 +24,25 @@ func newDevMount() v1.VolumeMount {
 	}
 }
 
-func NSMDPod(name string, node *v1.Node) *v1.Pod {
-	return createdNSMDPod(name, node, nil, nil)
+func newDevSrcMount() v1.VolumeMount {
+	return v1.VolumeMount{
+		Name:      "src",
+		MountPath: "/go/src",
```

Comment:
Please extract this path to a constant. Should we consider this an ENV variable?

Reply:
This is in sync with debug.sh, I will extract this changes to separate PR.